hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOMRatisSnapshots.java

```diff
@@ -347,6 +347,15 @@ private void checkSnapshot(OzoneManager leaderOM, OzoneManager followerOM,
     Path leaderActiveDir = Paths.get(leaderMetaDir.toString(), OM_DB_NAME);
     Path leaderSnapshotDir =
         Paths.get(getSnapshotPath(leaderOM.getConfiguration(), snapshotInfo));
+
+    // Get list of live files on the leader.
+    RocksDB activeRocksDB = ((RDBStore)leaderOM.getMetadataManager().getStore()).getDb().getManagedRocksDb()
+        .get();
+    List<String> liveSstFiles = new ArrayList<>();
+    // strip the leading "/".
+    liveSstFiles.addAll(activeRocksDB.getLiveFiles().files.stream().map(s -> s.substring(1)).collect(
+        Collectors.toList()));
```

Comment:
nit: can use the result of `collect()` directly, no need for two lists.

```suggestion
    // strip the leading "/".
    List<String> liveSstFiles = activeRocksDB.getLiveFiles().files.stream()
        .map(s -> s.substring(1))
        .collect(Collectors.toList());
```

(also, it can be a `Set`, since we don't need to consider duplicates)

Reply:
I also think it should be set for better search.